spec/tree_spec.rb

```diff
@@ -1,14 +1,66 @@
 require 'rspec'
 require 'tree'
 
+testTree = Tree.new
+
 describe 'Tree' do
```

Comment:
`describe Tree do` will work.

Then you can do stuff like `described_class.new.age!` or even `subject { described_class.new }`